src/app/shared/mocks/dspace-rest-v2/mocks/mock-response-map.ts

```diff
@@ -0,0 +1,10 @@
+import { InjectionToken } from '@angular/core';
+import mockSubmissionResponse from './mock-submission-response.json';
+
+export class MockResponseMap extends Map<string, any> {};
+
+export const MOCK_RESPONSE_MAP: InjectionToken<MockResponseMap> = new InjectionToken<MockResponseMap>('mockResponseMap');
+
+export const mockResponseMap: MockResponseMap = new Map([
+  // [ '/config/submissionforms/traditionalpageone', mockSubmissionResponse ]
```

Comment:
Is this meant to be commented out as an example? 

It'd be nice to add comments or Typedocs to this file to explain how this `MockResponseMap` works.  It's not self explanatory right now, and the commented out code looks a bit odd here.  However, I'm OK with this commented out example if it's explained via TypeDocs or similar.

Reply:
I added some comments to explain its purpose.